src/site/content/en/payments/web-based-payment-apps-overview/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Web-based payment apps developers guide
```

Comment:
```suggestion
title: Web-based payment apps overview
```
This is much more of an overview than a guide. Can we align the title with the URL?

Reply:
If I were to change this title, I would want to align the "[Android payment app developers guide](https://web.dev/android-payment-apps-overview/)" as well which isn't necessarily overview.

Reply:
Please explain your rationale about why it's more important for the URLs to be consistent than for the page to use the correct documentation type keyword (the correct keyword is `overview` but you want to use `developers guide`).

Here's my rationale about why it's important for the title to use the correct keyword. When I land on a page called `Overview`, I expect it to contain high-level conceptual information. Our audience (developers) has been trained to expect this because most technical documentation sites use the keywords `Overview`, `Guide`, `Tutorial`, and `Reference` consistently. When I land on a page called `Overview` but it is a step-by-step implementation guide, I am immediately frustrated because I just wanted high-level information about the topic but it is diving into way more detail than I need. When I land on a page called `Guide` but it is actually an overview, I am immediately frustrated because I needed information on how to implement but it's only providing high-level information. Furthermore, when I search for documentation, I also use one of they keywords to pinpoint what kind of documentation I need, e.g. `service worker tutorial` or `service worker reference`. 

For the record I originally set the Android page's URL to `/android-payment-apps-developer-guide` (so that the title would be consistent with the URL) and then you asked to change it to `/android-payment-apps-overview` in #3023, which I also disagreed with.

Reply:
- I think it's ok to change the title to "Web-based payment apps overview".
- The Android article currently titled "developers guide" but the URL has `overview` which was intentional. I thought I would split the article but it's unlikely happening and now I think it's ok to make it `developers-guide`.
- If we make this change I will add a reverse redirect (`overview` -> `developers-guide`), but when I add a new "Android payment app overview" article, I will remove the redirect.

Reply:
I think I made expected changes.